scripts/_common.sh

```diff
@@ -55,6 +55,15 @@ function run_test_for_versions() {
     export FM_OFFLINE_NODES=1
   fi
 
+  if \
+    [ "$fed_version" != "current" ] ||
+    [ "$client_version" != "current" ] ||
+    [ "$gateway_version" != "current" ] ; then
+    export FM_RUN_TEST_VERSIONS="$fed_version $client_version $gateway_version"
+  else
+    unset FM_RUN_TEST_VERSIONS
```

Comment:
Was this env var set before elsewhere or are you just unsetting it to avoid confusion if it's ever set by accident?

Reply:
With the current setup, I don't believe this env var will ever be set since each call to `run_test_for_versions` is in `parallel`, so each job should be setting `FM_RUN_TEST_VERSIONS` for the first time.

I'm a fan of keeping though since it removes a potential footgun.

Reply:
I think it's not needed, but explit here doesn't hurt, and I'm paranoid.